From 5c41b1a49bbacd3a1eb0ca08a50eb9573bdd5380 Mon Sep 17 00:00:00 2001
From: Gerion Entrup <gerion.entrup@flump.de>
Date: Wed, 13 Jan 2021 22:44:09 +0100
Subject: [PATCH] CMake: prevent lib copies for package creation

---
 src/hyperiond/CMakeLists.txt | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/hyperiond/CMakeLists.txt b/src/hyperiond/CMakeLists.txt
index 02bce862..57e26a95 100644
--- a/src/hyperiond/CMakeLists.txt
+++ b/src/hyperiond/CMakeLists.txt
@@ -147,10 +147,10 @@ if(WIN32)
 endif()
 
 # Deploy all dependencies for package creation (not for OSX)
-include(${CMAKE_SOURCE_DIR}/cmake/Dependencies.cmake)
-
-if (NOT ENABLE_OSX AND NOT WIN32) # Unix
-	DeployUnix("hyperiond")
-elseif(WIN32) # Windows
-	DeployWindows("hyperiond")
-endif ()
+# include(${CMAKE_SOURCE_DIR}/cmake/Dependencies.cmake)
+#
+# if (NOT ENABLE_OSX AND NOT WIN32) # Unix
+# 	DeployUnix("hyperiond")
+# elseif(WIN32) # Windows
+# 	DeployWindows("hyperiond")
+# endif ()
-- 
2.26.2

